src/attpc_engine/detector/writer.py

```diff
@@ -3,6 +3,7 @@
 
 import numpy as np
 import h5py as h5
+import os
```

Comment:
Do we need this import or can we remove it?

Reply:
Not needed and can be removed